src/system/Toolbar/Logo.tsx

```diff
@@ -0,0 +1,40 @@
+/** @jsxImportSource theme-ui */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+
+const VIP_LOGO = 'vip-logo-component';
+
+type LogoProp = {
+	className?: string;
```

Comment:
I really think we should stop accepting `className` prop unless there is a really well-defined use case that is explained in stories. It's not even clear which element the className gets applied to, it makes no sense.

Reply:
@chriszarate Let's discuss this in P2/Slack. Otherwise, the PR will stay here forever in this discussion. Blame the classes, not the 'following-standard' PR 🙏

Reply:
https://a8c.slack.com/archives/C02T9LLBDLP/p1699363463056159